src/transformers/models/whisper/configuration_whisper.py

```diff
@@ -129,14 +129,17 @@ class WhisperConfig(PretrainedConfig):
             Begin of stream token id.
         eos_token_id (`int`, *optional*, defaults to 50257):
             End of stream token id.
+        ts_start_token_id (`int`, *optional*, defaults to 50363)
```

Comment:
Would suggest making this more verbose in the style of `transformers` 🤗
```suggestion
        timestamp_token_id (`int`, *optional*, defaults to 50363)
```